docs/usage/controls-shortcuts.md

```diff
@@ -0,0 +1,38 @@
+## UI
+
+| Shortcut                       | Description                                                                                |
+| ------------------------------ | ------------------------------------------------------------------------------------------ |
+| Left Mouse Click               | Add selected waypoint from navbar at click position                                        |
+| Left Mouse Drag                | Panning around the map                                                                     |
+| Scroll Wheel, Trackpad gesture | Zoom in or out of the map                                                                  |
+| Delete/Backspace               | Delete highlighted waypoint                                                                |
+| Ctrl/⌘ + Z                     | Undo                                                                                       |
+| Ctrl/⌘ + Shift + Z, Ctrl/⌘ + Y | Redo                                                                                       |
+| Ctrl/⌘ + N                     | Creates a new File                                                                         |
+| Left Arrow                     | Highlights previous waypoint                                                               |
+| Right Arrow                    | Highlights next waypoint                                                                   |
+| G, Ctrl/⌘ + G                  | Generates current path                                                                     |
+| F                              | Add selected waypoint from navbar at last waypoint location. Defaults to coordinate (5, 5) |
+| 1 through 4                    | Selects waypoint type in navbar (2 selects 2nd waypoint type, Translation Waypoint)        |
+
+## Waypoint Rotation
+
+> NOTE: Holding Shift changes rotation to π/4. For example, Shift + E rotates selected waypoint by -π/4 (clockwise)
```

Comment:
Material for MkDocs has built-in support for these kinds of notes, called [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/).

In addition to being semantically correct in the Markdown source, they also look much better.

Reply:
Perfect! I'll do this for the blockquote above as well.